src/test/java/lotto/model/MoneyTest.java

```diff
@@ -0,0 +1,33 @@
+package lotto.model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class MoneyTest {
+
+    @Test
+    void 음수_돈_불가() {
+        assertThatThrownBy(() -> {
+            new Money(-1);
+        }).isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining("금액은 0 이상이어야 합니다.");
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"15000:15", "1234:1", "10:0", "190000:190"}, delimiter = ':')
+    void 구매_횟수(int amount, int buyCount) {
+        Money money = new Money(amount);
+        assertThat(money.buyChance(1000)).isEqualTo(buyCount);
+    }
+
+    @Test
+    void 음수가격_구매_불가() {
+        assertThatThrownBy(() -> new Money(1000).buyChance(0)).isInstanceOf(IllegalArgumentException.class)
```

Comment:
예외 검증시엔 대상 메서드만 전달해도 좋을 것 같아요.
```suggestion
        assertThatThrownBy(() -> money.buyChance(0)).isInstanceOf(IllegalArgumentException.class)
```